pyproject.toml

```diff
@@ -46,6 +46,13 @@ warn_return_any = false
 warn_unreachable = true
 pretty = true
 
+[[tool.mypy.overrides]]
+module = ["darwin.*", "tests.*"]
+disallow_any_unimported = false
+disallow_any_explicit = false
+disallow_untyped_calls = false
```

Comment:
What was it that necessitated these two (☝🏻 and 👇🏻), because this is not ideal to disallow globally, but I'm guessing you encountered something significant to change these flags.

Reply:
So the thing here is that these cause numpy and any third party library use to error if the third party doesn't have typing. To me there's no use being informed about errors we can't change, it just clogs up the visual and makes it harder to distinguish from things we can resolve.